src/Cache/Value.php

```diff
@@ -20,7 +20,7 @@ class Value
 	/**
 	 * Cached value
 	 */
-	protected $value;
+	protected mixed $value;
```

Comment:
I'm not entirely sure anymore but I believe that we had issues with mixed at some point and then decided to leave it untyped if the type is mixed.

Reply:
Earlier we didn't add it as it forces to also add it without much (any) typesafty earned. However, @lukasbestle brought up a while ago that we might still want to start adding it slowly for consistency and clarity (this really should be any type, not just we didn't bother to think through what types).

Reply:
I agree. In our first try, we went too far and added it to a lot of places at once. But I think e.g. in this place it's completely accurate as Nico wrote.